librz/core/cfile.c

```diff
@@ -1173,10 +1173,12 @@ RZ_API RzCoreFile *rz_core_file_open(RzCore *r, const char *file, int flags, ut6
 		goto beach;
 	}
 
-	fh = core_file_new(r, fd->fd);
 	if (!fh) {
-		eprintf("core/file.c: rz_core_open failed to allocate RzCoreFile.\n");
-		goto beach;
+		fh = core_file_new(r, fd->fd);
+		if (!fh) {
+			RZ_LOG_ERROR("core/file.c: rz_core_open failed to allocate RzCoreFile.\n");
```

Comment:
Even the filename is wrong, probably better to remove it from the error message,